rsocket-core/src/main/java/io/rsocket/resume/ResumeToken.java

```diff
@@ -0,0 +1,50 @@
+package io.rsocket.resume;
+
+import io.netty.buffer.ByteBufUtil;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.UUID;
+
+public final class ResumeToken {
+  private final byte[] resumeToken;
```

Comment:
Should this be stored as a bytebuffer? Seems like you'll save some allocations then.